File: CPP/9PalindromeNumber.cpp
#include<iostream>
#include<cstring>

using namespace std;

class Solution {
public:
    bool isPalindrome(int x) {
    	string tmp="";
    	if(x<0){
    		return false;
    	}
		while(x!=0){
			tmp += char(x%10+'0');
			x /=10;
		}
		for(int i=0; i<tmp.size()/2; i++){
			if(tmp[i]!=tmp[tmp.size()-1-i]){
				return false;
			}
		}
		return true;    
    }
};

int main(){
	
	return 0;
}

File: Python/9PalindromeNumber.py
__author__ = 'minyi'
#Determine whether an integer is a palindrome. Do this without extra space.


class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x<0:
            return False
        str_value = str(x)

        i=0
        while i<len(str_value):
            if str_value[i] != str_value[len(str_value)-1-i]:
                return False
            i += 1
        return True
